resources/sdk/clisdkclient/extensions/config/config.go

```diff
@@ -224,6 +224,19 @@ func SetLoggingEnabled(c Configuration, loggingEnabled bool) error {
 	}, &loggingEnabled)
 }
 
+func SetExperimentalFeature(profileName string, featureName string, enabled bool) error {
+	viper.Set(fmt.Sprintf("%s.%s_enabled", profileName, featureName), enabled)
+	return viper.WriteConfig()
+}
+
+func GetExperimentalFeature(profileName string, featureName string) bool {
+	err := viper.ReadInConfig()
+	if err != nil {
+		fmt.Println(err)
```

Comment:
One last suggestion, you could use logger.Error() to print this error. Also good to explicitly return false here.

Reply:
Do you mean I could import logger.go? I only see a Fatal method there. Or do you mean fmt.Errorf?

Reply:
Sorry, there's no error method. logger.Warn would be good here. It would mean that the error wouldn't be printed to the console if this function returns an error.
This function will return an error if no config file exists, which will usually be the case when people use the CLI in pipelines and we don't want this error message confusing things.